src/packages/emmett-mongodb/src/eventStore/mongoDBEventStore.ts

```diff
@@ -27,51 +30,100 @@ export type StreamNameParts<T extends StreamType = StreamType> = {
   streamId: string;
 };
 
-export type StreamToProject<EventType extends Event> = {
+export interface EventStreamProjection<
+  ShortInfoType extends DefaultRecord = DefaultRecord,
+> {
+  details: { name?: string };
+  short: ShortInfoType | null;
```

Comment:
**QUESTION:** Could you explain me why we still have short? property here? Is it needed? I think that we could just keep the regular map of object inside the projections property of stream. Could you explain why do we need this additional nesting?